src/scss/main.scss

```diff
@@ -560,6 +560,54 @@ table th {
     text-align: left;
 }
 
+/* TABLE OF EXPERIMENTS */
```

Comment:
something needs to be done to fix this ugly corner:

![2020-12-20-145505_149x115_scrot](https://user-images.githubusercontent.com/3043706/102715118-6120bc00-42d3-11eb-925c-98d22aeeef71.png)

Reply:
It does not look like that in my browser, at least in chrome in mozilla, any idea why?

![Screenshot 2020-12-20 at 18 55 12](https://user-images.githubusercontent.com/22526102/102720513-131cb000-42f5-11eb-879c-fb76f58c1f1d.png)